cmd/opm/index/prunestranded.go

```diff
@@ -36,6 +36,7 @@ func newIndexPruneStrandedCmd() *cobra.Command {
 	indexCmd.Flags().StringP("binary-image", "i", "", "container image for on-image `opm` command")
 	indexCmd.Flags().StringP("container-tool", "c", "podman", "tool to interact with container images (save, build, etc.). One of: [docker, podman]")
 	indexCmd.Flags().StringP("tag", "t", "", "custom tag for container image being built")
+	indexCmd.Flags().Bool("skip-tls", false, "skip TLS certificate verification for container image registries while pulling index")
```

Comment:
For future reference, if this option is shared by all subcommands, you can place it on the root command (see [persistent flags](https://github.com/spf13/cobra/blob/master/README.md#persistent-flags)).